spec/ruby/optional/capi/spec_helper.rb

```diff
@@ -119,6 +120,32 @@ def compile_extension_jruby_truffle(name)
   File.delete(sulong_config_file) if File.exist?(sulong_config_file)
 end
 
+def compile_jruby_truffle_extconf_make(name, path, objdir)
+  ext       = "#{name}_spec"
+  file      = "#{ext}.c"
+  source    = "#{path}/#{file}"
+  temp_dir = Dir.mktmpdir
```

Comment:
This should be right before the `begin`, otherwise it could leave an extra tmpdir if the lines below fail.
One way is moving those lines in the begin.